docs/user_guide/timeseries/forecasting/LagFeatures.rst

```diff
@@ -0,0 +1,257 @@
+.. _lag_features:
+
```

Comment:
@Morgan-Sell

Reply:
Looks good! I had one comment to make the text more concise.